content/en/getting_started/dashboards/_index.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Getting Started with Dashboards
+kind: documentation
+further_reading:
+- link: "https://learn.datadoghq.com/enrol/index.php?id=8"
+  tag: "Learning"
+  text: "Self-paced online learning: Building Better Dashboards"
+- link: "/dashboards/"
+  tag: "Documentation"
+  text: "Dashboards basics"
+- link: "/notebooks/"
+  tag: "Documentation"
+  text: "Tell a story about data with Notebooks"
+- link: "/monitors/"
+  tag: "Documentation"
+  text: "Monitors, SLOs, notifications, downtimes, and incidents"
+---
+
+## Overview
+
+The key to getting started with dashboards is knowing what kinds of questions you ask yourself regularly. What are common issues your customers face? When a problem occurs, what questions help you find a solution? 
+
+Creating a good dashboard is about bringing the answers to these questions to the surface. Also, it is important not to cram all of those conversations into the same dashboard. Create separate dashboards to show answers to different kinds of questions.
+
+This guide gets you started on a path to making dashboards. These basic dashboards will help your team talk about things that really matter and get to the bottom of problems quickly.
+
+## Prerequisites
+
+If you haven't already, create a [Datadog account][1]. Install an agent on a host, and an integration for something running on that host.
+
+## Plan
+
+Decide if your dashboard will be a timeboard (all graphs on the same time scale) or screenboard (wider variety of widgets, different time scales). Read more about [timeboards and screenboards][2] if you're not sure.
+
+Also think about what purpose you're creating the dashboard for. A dashboard can help you and your teammates focus on the right work. A _team dashboard_ reminds you what's high priority, what needs attention now, and what you're succeeding at. Make a team dashboard (or multiple) with the information that people most frequently need they have to dig for. SLO and SLI details make for an excellent team dashboard.
```

Comment:
```suggestion
Also, determine the purpose of the dashboard you're creating. A dashboard can help you and your teammates focus on the right work. A _team dashboard_ reminds you what's high priority, what needs attention now, and what you're succeeding at. Make a team dashboard (or multiple) with the information that people most frequently need they have to dig for. SLO and SLI details make for an excellent team dashboard.
```

Reply:
https://docs.datadoghq.com/dashboards/widgets/slo/ - would be good to link to the SLO widget here.